oak-search-elastic/src/test/java/org/apache/jackrabbit/oak/plugins/index/elastic/ElasticConnectionRule.java

```diff
@@ -186,4 +263,37 @@ private void setUseDocker(boolean useDocker) {
     public boolean useDocker() {
         return useDocker;
     }
+
+    public class ElasticConnectionConnectionModel {
```

Comment:
`Connection` appears twice

```suggestion
    public class ElasticConnectionModel {
```

Reply:
Done